logstash_transit/pipeline/udp.conf.template

```diff
@@ -0,0 +1,18 @@
+input {
+  udp {
+    port => 514
+  }
+
+  heartbeat {}
+}
+
+output {
+  elasticsearch {
+    hosts    => "${ES_HOST}"
+    user     => "${ES_USER}"
+    password => "${ES_PASS}"
+    ssl      => true
+  }
+
```

Comment:
You can add a property here:

`add_field => {"service_name" => "$SERVICE_NAME"}` - but then you would have to make sure that the service has this set.

Reply:
actually this won't work as we are having 1 log service per cluster, so this needs to be dynamic.